site/en/docs/extensions/mv2/messaging/index.md

```diff
@@ -308,43 +308,42 @@ native app. For more examples and for help in viewing the source code, see [Samp
 [1]: #simple
 [2]: #connect
 [3]: #external
-[4]: /docs/extensions/runtime#method-sendMessage
-[5]: /docs/extensions/tabs#method-sendMessage
-[6]: /docs/extensions/runtime#event-onMessage
-[7]: /docs/extensions/runtime#method-connect
-[8]: /docs/extensions/tabs#method-connect
-[9]: /docs/extensions/runtime#type-Port
-[10]: /docs/extensions/tabs#method-connect
-[11]: /docs/extensions/runtime#event-onConnect
-[12]: /docs/extensions/runtime#type-Port
-[13]: /docs/extensions/tabs#method-connect
-[14]: /docs/extensions/runtime#method-connect
-[15]: /docs/extensions/runtime#method-connectNative
-[16]: /docs/extensions/runtime#type-Port
-[17]: /docs/extensions/runtime#property-Port-postMessage
-[18]: /docs/extensions/tabs#method-connect
-[19]: /docs/extensions/runtime#event-onConnect
-[20]: /docs/extensions/runtime#method-connect
-[21]: /docs/extensions/runtime#property-Port-onDisconnect
-[22]: /docs/extensions/runtime#event-onConnect
-[23]: /docs/extensions/runtime#event-onConnect
-[24]: /docs/extensions/runtime#property-Port-disconnect
-[25]: /docs/extensions/runtime#event-onMessageExternal
-[26]: /docs/extensions/runtime#event-onConnectExternal
+[4]: /docs/extensions/reference/runtime#method-sendMessage
+[5]: /docs/extensions/reference/tabs#method-sendMessage
+[6]: /docs/extensions/reference/runtime#event-onMessage
+[7]: /docs/extensions/reference/runtime#method-connect
+[8]: /docs/extensions/reference/tabs#method-connect
+[9]: /docs/extensions/reference/runtime#type-Port
+[10]: /docs/extensions/reference/tabs#method-connect
+[11]: /docs/extensions/reference/runtime#event-onConnect
+[12]: /docs/extensions/reference/runtime#type-Port
+[13]: /docs/extensions/reference/tabs#method-connect
+[14]: /docs/extensions/reference/runtime#method-connect
+[15]: /docs/extensions/reference/runtime#method-connectNative
+[16]: /docs/extensions/reference/runtime#type-Port
+[17]: /docs/extensions/reference/runtime#property-Port-postMessage
+[18]: /docs/extensions/reference/tabs#method-connect
+[19]: /docs/extensions/reference/runtime#event-onConnect
+[20]: /docs/extensions/reference/runtime#method-connect
+[21]: /docs/extensions/reference/runtime#property-Port-onDisconnect
```

Comment:
```suggestion
[21]: /docs/extensions/reference/runtime#property-Port
```

Seems we don't have deep links to properties on types. Linking to the type is as close as we can get.